src/runtimes/docker/contracts/ContainerClient.ts

```diff
@@ -4,6 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import type { FileType, ShellQuotedString } from 'vscode';
+import { DockerClientInfo } from '../clients/DockerClientBase/DockerInfoRecord';
```

Comment:
Don't import directly from the client types; any types referenced in ContainerClient should be defined in this file.

Reply:
Will do. But is there any particular reason why we do this?